lib/index.js

```diff
@@ -74,17 +74,10 @@ export default {
 
         // Call ERB to "de-templatize" the code
         return helpers.exec(this.erbPath, erbArgs,
-          { stdin: text, cwd: fileDir }
+          { stdin: text.replace(/<%=/g, '<%'), cwd: fileDir }
```

Comment:
Right, good call. I thought the rails scope was also `text.html.erb`. Are you sure in the case of rails it's being detected as `text.html.ruby`? The bottom right indicator says "HTML (Ruby - ERB)" and I see no other ERB scope in the list. In fact, how would Atom tell Rails ERB from regular ERB? Extension is the same and the tags are the same. The only difference is that Rails ERB supports writing an incomplete expression inside `<$= %>` and regular ERB doesn't.